### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by GERIskillzZz** [foxfolio/foxfolio-desktop#5]
It would be great to add a functionality to add the buy-price of a coin, so it's possible to see the win/lose over time.

**Comment by mschrimpf** [comment]
Ideally the buy price could maybe even be retrieved automatically from the exchanges

**Comment by GERIskillzZz** [comment]
Yes, maybe with a Date-Time Selectbox. Cryptocompare's Portfolio works with manual input and automatical price retreiving from an Exchange/API.

**Comment by greimela** [comment]
Hey, thanks for the suggestion!
Profit calculation is something I want to integrate as soon as possible.

Ideally based on automatically pulling the trades using the exchange APIs.
But manually adding transactions is also required, e.g. when using services like Shapeshift.

I'll look into how this is done at Cryptocompare.

**Comment by tornadoatc** [comment]
You might be able to leverage A.L.E to effectively generate a ledger for each coin at each exchange: https://github.com/CjS77/ale  

Knowing you average cost per coin would be very helpful!
